design-proposals/migration-target.md

```diff
@@ -0,0 +1,177 @@
+# Overview
+We are getting asks from multiple cluster admin that would like to explicitly specify the "destination" of the VM when doing Live migration.
+While this may be less important in a cloud-native environment,
+we get this ask from many users coming from other virtualization solutions, where this is a common practice.
+The same result can already be achieved today with a few steps, this is only about simplifying it with a single direct API on the single `VirtualMachineInstanceMigration` without the need to alter a VM spec.
+
+## Motivation
+In the ideal cloud native design, the scheduler is supposed to be always able to correctly identify
+the best node to run a pod (so the target pod for the VMI after the live-migration) on.
+In the real world, we still see specific use cases where the flexibility do explicitly and directly define the target node for a live migration is a relevant nice-to-have:
+- Experienced admins are used to control where their critical workloads are move to 
+- Workload balancing solution doesn't always work as expected
+- Troubleshooting a node
+- Validating a new node migrating there a specific VM
+
```

Comment:
We should also mention, that besides admins asking for _this_ feature, admins are also asking for, for example descheduler integration (see the proposal about deschedler intergratoin), load ware scheduling (look for link to kubevirt and 2nd schedulers).
What I want to say: This feature has to be seen in the context of all scheduling related features that we are currently adding (namely descheduler, and 2ndary scheduler is very old integration from long ago).

Again: This is the tool for expcetions, not the norm.